dts/msm8916/msm8939-xiaomi-ido.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <0xef 0x30000 0xf1 0x30000 0x107 0x30000>;
+	qcom,board-id = <0x1000b 0xb 0x2000b 0xb>;
```

Comment:
Consider changing turning this into:
`qcom,board-id = <0x1000b 11>, <0x2000b 11>;`